tests/unit/Pelagos/Util/ISOMetadataExtractorUtilTest.php

```diff
@@ -299,6 +299,8 @@ public function setUp()
                 'getMetadataContacts' => new ArrayCollection(array($this->mockPersonDatasetSubmissionMetadataContact)),
                 'getDistributionPoints' => new ArrayCollection(array($this->mockDistributionPoint)),
                 'getErddapUrl' => 'https://xyz',
+                'getDatasetFileUrlLastCheckedDate' => new \DateTime('now', new \DateTimeZone('UTC')),
+                'getDatasetFileUrlStatusCode' => '200'
```

Comment:
I think we should ideally leave these null, as the attributes are now nullable. At this point in the process, a user is making a submission, which happens well before we mark things are remotely hosted.